frontend/public/components/catalog/catalog-items.tsx

```diff
@@ -1,9 +1,14 @@
 import * as React from 'react';
 import * as _ from 'lodash-es';
+import * as PropTypes from 'prop-types';
```

Comment:
Linter error:
```
ERROR in /Users/rhamilto/Git/console/frontend/public/components/catalog/catalog-items.tsx(3,1):
TS6133: 'PropTypes' is declared but its value is never read.
```

Reply:
I removed the Proptypes import. (Came in through the rebase.)